src/docfx/config/ops/OpsConfigLoader.cs

```diff
@@ -82,7 +85,17 @@ internal static class OpsConfigLoader
                     ["open_to_public_contributors"] = docsetConfig.OpenToPublicContributors,
                 };
 
-                result["splitTOC"] = JArray.FromObject(docsetConfig.SplitTOC);
+                var splitTOCSet = docsetConfig.SplitTOC ?? new HashSet<PathString>();
+
+                foreach (var item in splitTOCOutOfDocsetsToPublish)
```

Comment:
nit: This variable seems unneeded
```suggestion
                foreach (var item in opsConfig.SplitTOC)
```